From 48075c86543d74f7d1d9e089ad255d9935314c92 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 13 May 2020 15:53:20 +0300
Subject: [PATCH] [SW-0] preboot: remove TARGET_IS_TESTER

The TARGET_IS_TESTER was used in Goya1 and is not used anymore.
Remove it.

Change-Id: I9bd8aaf80f1ef3b0f5679547b999c0b58544e843
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 3598496..ff5b9e9 100644
--- a/Makefile
+++ b/Makefile
@@ -12,9 +12,6 @@
 ifeq ($(HL_TARGET),pldm)
 	CFLAGS += -DTARGET_IS_PLDM
 endif
-ifneq ($(TARGET_IS_TESTER),)
-	CFLAGS += -DTARGET_IS_TESTER
-endif
 
 CFLAGS += -Os -fdata-sections -ffunction-sections
 SFLAGS := $(CFLAGS) -D__ASSEMBLY__
diff --git a/app/preboot.c b/app/preboot.c
index 403dcea..a4d5253 100644
--- a/app/preboot.c
+++ b/app/preboot.c
@@ -117,7 +117,6 @@
 	}
 }
 
-#ifndef TARGET_IS_TESTER
 static const char *preboot_ver_str = "Preboot " VERSION_STR;
 static char preboot_btl_ver_str[VERSION_MAX_LEN] = "BTL version ";
 
@@ -154,9 +153,6 @@
 
 	regmap_writel((u32)(offset & ~SDRAM_BASE), mmPREBOOT_VER_OFFSET);
 }
-#else	/* TARGET_IS_TESTER */
-__attribute__((weak)) void preboot_version(void) {}
-#endif	/* TARGET_IS_TESTER */
 
 void preboot_main(void)
 {
